inputs/worker_inner:6.json

```diff
@@ -111,10 +95,7 @@
   ],
```

Comment:
Let's remove `"verify_ssl"`, `"koji_keytab"`, and `"koji_principal"` from `koji_upload`

Reply:
Fixed I think.